mitosheet/mitosheet/tests/step_performers/test_merge.py

```diff
@@ -598,15 +598,6 @@ def test_incompatible_merge_key_types_error():
 
     assert mito.transpiled_code == []
 
-
-def test_merge_between_multi_index_and_non_errors():
-    df_one = make_multi_index_header_df({0: [1, 2], 1: [3, 4]}, ['A', ('B', 'count')])
-    df_two = pd.DataFrame({'A': [1, 2], 'B': [5, 6]})
```

Comment:
Why are you removing this test?

Reply:
We don't actually support multi-index headers currently, this was legacy from when we tried to do so.